requirements.txt

```diff
@@ -1,5 +1,5 @@
-Flask < 2.0.0
-Flask-Injector
+uvicorn
+fastapi
 PyJWT
 PyYAML
 Werkzeug < 2.0.0
```

Comment:
Werkzeug is a dependency of Flask and unless this is used by FastAPI for something (I assume its not) this can be removed too.